pandera/schemas.py

```diff
@@ -163,6 +163,12 @@ def _check_dataframe(self, dataframe):
                     check.prepare_dataframe_input(dataframe)))
         return all(val_results)
 
+
+    @property
+    def dtype(self) -> Dict[str, str]:
```

Comment:
do you think this should match the `pd.DataFrame` API?

Reply:
@cosmicBboy Sorry, I misread your earlier comment on the issue about naming `DataFrameSchema.dtypes`. 

I could see an argument in either direction on this one. `pd.DataFrame.dtypes` returns a `pd.Series` with the column types. `pd.DataFrame.astype` and `pd.read_csv` take a property named `dtype` which expects a `dict` not a `pd.Series`.

In either case we'll want to return a `dict`. I had originally named it `dtype` since that is the parameter name used in the methods which expect the `dict`. However, I'm happy to switch if you prefer going with `dtypes`.

Reply:
hmm, yeah I think in this case going for the `dict` is the way to go, and now I'm going back and forth between `dtypes` and `dtype` for the DataFrameSchema... I think for now let's stick to `dtype` for both DataFrameSchema and SeriesSchema, since the use case is to use this property in the `as_type` and `read_{csv, json, *}` methods.